_includes/manuals/1.24/5.7.7_openid_sso.md

```diff
@@ -0,0 +1,105 @@
+### Single Sign-on for Foreman using OpenID Connect protocol
+
+An OpenID Provider(OP)  implements a Single Sign-on (SSO) using  an OpenID Connect (OIDC) protocol. OIDC verifies the identity of the End-user depending on the given authentication details. In this example, [Keycloak](https://www.keycloak.org/) is used as the IDP, and Foreman is the Relying Party (RP)
```

Comment:
nit: s / `https://keycloak.org` / `https://keycloak.example.com`

Reply:
Actually, I think this is correct since it's referring to the project. There are other OIDC providers.

Reply:
oh! my bad.